### DISCUSSION TITLE
Add Record button in detail view

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15677] [How to] [answered]
In frequent data entry, it is necessary to get back to list view to add record. Would be helpful to add relevant button in detail view, or extend shortcuts as per #15466

Best

**Comment by mariuszkrzaczkowski** [comment]
it works now
https://yetiforce.com/en/knowledge-base/documentation/user-documentation/item/keyboard-shortcuts

**Comment by rskrzypczak** [comment, marked as answer]
> Would be helpful to add relevant button in detail view

This link is already in breadcrumbs :)
![image](https://user-images.githubusercontent.com/10221999/151748291-07afe31c-7d41-4a63-9654-2545e743c3c6.png)

**Comment by JohnMaverick** [comment]
crumbs have to be hidden here due to profile design

**Reply by JohnMaverick** [reply]
But shortcut is cool anyway, thank for touching subject. Only easy config of shortucts would be helpfull :-)
